src/components/HeaderWithRulers/HeaderWithRulers.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import styled from '@emotion/styled'
+
+interface Props {
+  text: string
```

Comment:
think this should use `children` instead of using a `text` prop. As when you using a `h` tag, you create one like this:
```
<h1>heading</h1>
```
instead of this
```
<h1 text='heading' />
```